addons/edi_notifier/models/email_notifier.py

```diff
@@ -17,8 +17,12 @@ def _should_notify(self, notifier, event_type, rec):
 
     @api.multi
     def _notify(self, notifier, event_type, recs):
+        instance_url = self.env['ir.config_parameter'].sudo().get_param('web.base.url')
```

Comment:
Nit: `self.env['ir.config_parameter']` should be put into its own variable at the top of the method, e.g.:

`ConfigParameter = self.env["ir.config_parameter"]`